homeassistant/components/homematicip_cloud/__init__.py

```diff
@@ -53,7 +53,7 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
             for entry in hass.config_entries.async_entries(DOMAIN)
         }:
             hass.async_add_job(
-                hass.config_entries.flow.async_init(
+                hass.config_entries.flow.async_init(  # type: ignore
```

Comment:
What's the type error we're ignoring? Please specify it:
```py
# type: ignore[bla-bla]
```

Reply:
Done [arg-type]
homeassistant/components/homematicip_cloud/__init__.py:56: error: Argument 1 to "async_add_job" of "HomeAssistant" has incompatible type "Coroutine[Any, Any, FlowResult]"; expected "Callable[..., Any]"  [arg-type]

Seems like async_add_job is too limited, but changing that is not part of this PR.